example/cordova/www/css/icons.css

```diff
@@ -0,0 +1,70 @@
+/* Material Icons Font (for MD theme) */
```

Comment:
Let's put that to folder that will allow us to update it etc. For the moment it's not clear where this CSS comes from. Angular Material? It should be clear how to update that.
We could also try to use npm install to distribute that etc.